website/docs/add-secure-apps/providers/rac/rac-public-key.md

```diff
@@ -0,0 +1,108 @@
+---
+title: RAC SSH Public Key Authentication
+---
+
+## About RAC SSH public key authentication
+
+The RAC provider supports SSH public key authentication. This allows for secure connections to SSH endpoints without the use of passwords.
+
+SSH private keys can be configured via several methods:
```

Comment:
Is this the same info as in file above, on line 57 ("RAC connection settings can be set via several methods:" ...?

Ideally we would not repeat the same exact chunk of content, or if we absolutely need to then we cold choose (if worth the effort) to use a "snippet" that lives in one place (probably [here](https://github.com/goauthentik/authentik/tree/main/website/docs/expressions) and gets used wherever we want.